app/scripts/metamask-controller.js

```diff
@@ -2019,11 +2019,13 @@ export default class MetamaskController extends EventEmitter {
         'AccountsController:listAccounts',
         'AccountsController:getSelectedAccount',
         'AccountsController:setSelectedAccount',
+        'NetworkController:getState',
+        'NetworkController:setActiveNetwork',
       ],
     });
 
     this.mmiController = new MMIController({
-      messenger: mmiControllerMessenger,
+      messagingSystem: mmiControllerMessenger,
       mmiConfigurationController: this.mmiConfigurationController,
       keyringController: this.keyringController,
       preferencesController: this.preferencesController,
```

Comment:
As you have removed preferencesController from MMIController options can we remove it here too ?